tests/test_center_calculation.py

```diff
@@ -0,0 +1,53 @@
+import json
+from unittest import TestCase
+
+from process_subways import calculate_centers
+from subway_io import load_xml
+
+
+class TestCenterCalculation(TestCase):
+    """Test center calculation. Test data [should] contain among others
+    the following edge cases:
+      - an empty relation. It's element should not obtain "center" key.
+      - relation as member of relation, the child relation following the parent
+        in the OSM XML file.
+      - relation with incomplete members (broken references).
+      - relations with cyclic references.
+    """
+
+    OSM_DATA = "tests/assets/kuntsevskaya_transfer.osm"
```

Comment:
Relative paths will make it easier to run tests with PyCharm runner
```python
    ASSETS_PATH = Path(__file__).resolve().parent / "assets"
    OSM_DATA = str(ASSETS_PATH / "kuntsevskaya_transfer.osm")
    CORRECT_CENTERS = ASSETS_PATH / "kuntsevskaya_centers.json"
```

Reply:
\+